tests/test_version.py

```diff
@@ -5,5 +5,25 @@
 
 def test_version():
     ver = get_git_version(abbrev=7)
-    VER_RE = r"^(\d+.\d+.\d+)(-\d+)*(-g\S{7})?(-dirty)?$"
+    VER_RE = re.compile(
```

Comment:
@ali-tny could you add a comment/link to `semver` project in case we need to revisit this ever

Reply:
👍